pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -352,8 +352,8 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 
 	dataDir := k8sutil.DataDir
 	// Create volume config for /dev so the pod can access devices on the host
-	// Only valid when running OSD with LVM and Raw mode
-	if !osdProps.onPVC() {
+	// Only valid when running OSD on device or OSD on LV-backed PVC
+	if !osdProps.onPVC() || osd.CVMode == "lvm" {
```

Comment:
is LV-backed PVC using ceph-volume lvm mode or raw mode?

Reply:
Currently, it uses raw mode. It used lvm mode in the past. This PR adds the missing "/dev/" mount in the latter case.

Reply:
`osd.CVMode == "lvm"` is only set for new OSDs when they are encrypted, or when metadata/wal devices are used, right? And otherwise it is `"raw"`? If so, LGTM

Reply:
Plus, and is also set for old OSDs like. Then CVMde is "" and is set to "lvm" before this condition.

https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/osd/osd.go#L516